content/en/docs/user-guides/how-to/plugin-management.md

```diff
@@ -11,22 +11,72 @@ Plugins for `notation` provide functionality and integration with key stores and
 
 ## Install a plugin
 
-To install a plugin, download the plugin binary, create a `{plugin-name}` directory in `{NOTATION_LIBEXEC}/plugins/`, and add the plugin binary to that directory. Alternatively, if you are using an application that bundles a plugin and the `notation` binary together, such as [AWS Signer](https://docs.aws.amazon.com/signer/latest/developerguide/image-signing-prerequisites.html), see the installation instructions from the vendor.
+To install a plugin, run the `notation plugin install` command to directly install a plugin either from a URL or from the host file system. This will create a `{plugin-name}` directory in `{NOTATION_LIBEXEC}/plugins/` if the directory does not exist. The supported plugin file formats are `.zip, .tar.gz` and `single plugin executable file`. Alternatively, if you are using an application that bundles a plugin and the `notation` binary together, such as [AWS Signer](https://docs.aws.amazon.com/signer/latest/developerguide/image-signing-prerequisites.html), see the installation instructions from the vendor.
 
 {{% alert title="Warning" color="warning" %}}
 Before creating the `{plugin-name}` directory, confirm you are using a name that follows the [naming structure](#plugin-naming-structure) for plugins. Plugins that do not follow the naming structure will not be recognized by `notation`.
 {{% /alert %}}
 
-The following example downloads and installs version 0.6.0 of [notation-azure-kv](https://github.com/Azure/notation-azure-kv) plugin for on macOS with Apple Silicon using the default location for the Notation installation.
+## Usage
 
-{{% alert title="Warning" color="warning" %}}
-The following example only works for version 0.6.0 of the *notation-azure-kv* plugin on macOS with Apple Silicon using the default location for the Notation installation. You will need to update the filenames, location, and commands for other plugins, versions, and platforms. For more details on the default location of that directory on each platform, see [Notation directory structure for system configuration]({{< ref "/docs/user-guides/how-to/directory-structure.md" >}}).
-{{% /alert %}}
+### Install a plugin from URL:
+
+`notation plugin install --file <file_path>`
+
+### Install a plugin from file system: 
+
+`notation plugin install --sha256sum <digest> --url <HTTPS_URL>`
+
+The following examples show how to install each plugin:
```

Comment:
The following examples show how to install each plugin on a Linux AMD64 machine. For Windows and macOS, please get the URL or plugin binary from plugin vendors' website.